packages/arb-token-bridge-ui/src/components/TransferPanel/TransferPanelSummary.tsx

```diff
@@ -1,197 +1,14 @@
-import React, { useEffect, useMemo, useState } from 'react'
-import { BigNumber, constants, utils } from 'ethers'
+import React from 'react'
 import { InformationCircleIcon } from '@heroicons/react/24/outline'
-import { useLatest } from 'react-use'
 
 import { Tooltip } from '../common/Tooltip'
 import { useAppState } from '../../state'
 import { useETHPrice } from '../../hooks/useETHPrice'
-import { useDebouncedValue } from '../../hooks/useDebouncedValue'
+
 import { formatAmount, formatUSD } from '../../util/NumberUtils'
 import { isNetwork } from '../../util/networks'
 import { useNetworksAndSigners } from '../../hooks/useNetworksAndSigners'
-import { tokenRequiresApprovalOnL2 } from '../../util/L2ApprovalUtils'
-import { useGasPrice } from '../../hooks/useGasPrice'
-
-export type GasEstimationStatus = 'idle' | 'loading' | 'success' | 'error'
```

Comment:
Not sure why we moved all this code to a separate file. It was only used here, so extracting to common `hooks` folder doesn't make much sense to me. Makes the diff much harder to figure out.

Reply:
Actually `useGasSummary` was defined here but it wasn't even being used here. It was being exported and used in `TransferPanel.tsx`. So I took this as an opportunity to give it it's own place since this contains a lot of business logic and we might want to isolate it from tsx.

Reply:
Okay, to minimize diff I can revert it back to where it was. I still think it isn't where it should ideally be.

Reply:
Done

Reply:
@fionnachan I would keep moving the function/hook location in a separate, more directed PR. 
Since we are also changing the internal code of the function in this PR, I agree with Dragisa to give priority to keeping diffs minimum so that the review is easy and potential problems are surfaced easily.

Reply:
yea, my comment wasn't a change request. it was a reply to your conversation regarding where it should be located at